src/deepsparse/loggers/function_logger.py

```diff
@@ -80,6 +80,9 @@ def log(self, identifier: str, value: Any, category: MetricCategories):
         if extracted_value != NO_MATCH:
             if self._function_call_counter % self.frequency == 0:
                 mapped_value = self.function(extracted_value)
+                identifier = (
```

Comment:
We need to include the remainder into the identifier, otherwise, the logs may end up confusing the user.

Reply:
as discussed with @corey-nm not needed